doc/src/fix_restrain.rst

```diff
@@ -17,10 +17,18 @@ Syntax
 
   .. parsed-literal::
 
-       *bond* args = atom1 atom2 Kstart Kstop r0
+       *bond* args = atom1 atom2 Kstart Kstop r0start r0stop
```

Comment:
Please put (r0stop) in parens.  And below say something like
... end of run (optional, set to r0start if not used) (distance units)
Ditto for lblond